packages/tools/bcm2835-bootloader/package.mk

```diff
@@ -48,7 +48,10 @@ makeinstall_target() {
     [ -f dt-blob.bin ] && cp -PRv dt-blob.bin $INSTALL/usr/share/bootloader/dt-blob.bin
 
     cp -PRv $PKG_DIR/scripts/update.sh $INSTALL/usr/share/bootloader
-    
+
+    if [ -f $DISTRO_DIR/$DISTRO/config/distroconfig.txt ]; then
+      cp -PRv $DISTRO_DIR/$DISTRO/config/distroconfig.txt $INSTALL/usr/share/bootloader
```

Comment:
I've tested, and including `distroconfig.txt` when it doesn't exist is fine, it just results in a few entries appearing in `vcdb log msg`:
```
rpi22:~ # vcdbg log msg 2>&1 | grep distroconfig.txt
001121.664: Failed to open include file 'distroconfig.txt'
001169.870: Failed to open include file 'distroconfig.txt'
001907.108: Failed to open include file 'distroconfig.txt'
```
What we could do here is add an empty (zero byte) `$PKG_DIR/files/3rdparty/bootloader/distroconfig.txt` to this PR, then add
```
    else
      cp -PRv $PKG_DIR/files/3rdparty/bootloader/distroconfig.txt $INSTALL/usr/share/bootloader
```

Not strictly necessary of course, the errors are harmless (and hopefully stay that way) but would be a more complete solution.

Minor nit: could you add a blank line between lines 54 and 55?